code/cmdline/cmdline.cpp

```diff
@@ -382,6 +382,7 @@ cmdline_parm use_warp_flash("-warp_flash", NULL, AT_NONE);	// Cmdline_warp_flash
 cmdline_parm allow_autpilot_interrupt("-no_ap_interrupt", NULL, AT_NONE);
 cmdline_parm stretch_menu("-stretch_menu", NULL, AT_NONE);	// Cmdline_stretch_menu
 cmdline_parm no_screenshake("-no_screenshake", nullptr, AT_NONE); // Cmdline_no_screenshake
+cmdline_parm deadzone("-deadzone", "Sets the joystick deadzone. Integer value from 0 to 100. Disables deadzone slider in the in-game Options menu.", AT_INT); //Cmdline_deadzone
```

Comment:
Should this save `Overrides deadzone slicer` instead of `Disables`? `Disables` makes it sound like the buttons on the options screen will not even show up.

Reply:
Its disabled in the same way that the difficulty slider is disabled, the background image is there, but the dots are either all black or otherwise won't accept input from the user.

Reply:
Recommend stating somewhere in the description that the max deadzone size is 50% of the joystick range.